pages/learn/getting-started-step-by-step.md

```diff
@@ -99,28 +99,32 @@ Using the product catalog example, `productId` is a numeric value that uniquely
 To add the `properties` object to the schema:
 
 1. Add the `properties` validation keyword to the end of the schema:
-    
+
 ```json
-...
-"title": "Product",
```

Comment:
If we're going to make these syntax-highlighted, we need to do this differently.

    ```jsonc
    // ...
    "title": "Product",

`jsonc` allows comments, and I'm pretty sure we can get it to have the same highlight colors as `json` (probably in another PR if it's not already that way)